schema.graphql

```diff
@@ -170,6 +170,7 @@ scalar MAC
 
 type Mutation {
   addUser(email: String!, given_name: String!, family_name: String!): User
+  initialize(email: String!, given_name: String!, family_name: String!): User
```

Comment:
Adding context per our discussion. During initialization, the user should claim themselves as admin. 

For the secondary admin. Let's do it in another PR.